wagtailio/core/blocks.py

```diff
@@ -64,13 +66,31 @@ class Meta:
         template = "core/blocks/page_or_external_link_block.html"
 
 
+class BannerVideoBlock(media_blocks.AbstractMediaChooserBlock):
+    pass
```

Comment:
suggestion:

```suggestion
    @cached_property
    def field(self):
        return forms.ModelChoiceField(
            queryset=self.target_model.objects.filter(type="video"),
            widget=self.widget,
            required=self._required,
            validators=self._validators,
            help_text=self._help_text,
        )
```

to limit just to videos

Reply:
Thanks! Manually applied to add imports